src/Controller/EnMarche/ReferentController.php

```diff
@@ -65,13 +65,15 @@ public function usersSendMessageAction(Request $request): Response
             return $this->redirectToRoute('app_referent_users');
         }
 
-        $message = new ReferentMessage($this->getUser(), $filter);
```

Comment:
So now you don't need this var anymore and can just get it only when the form is valid.

Reply:
this `$message` is literally mapped to this form

Reply:
Sorry I was targeting `$messageNotifier` in the other column of the diff view.

Reply:
of course, good catch :+1:

Reply:
fixed